modules/nw-egress-ips-about.adoc

```diff
@@ -79,7 +79,7 @@ For clusters provisioned on public cloud infrastructure, there is a constraint o
 IP capacity = public cloud default capacity - sum(current IP assignments)
 ----
 
-While the Egress IPs capability manages the IP address capacity per node, it is important to plan for this constraint in your deployments. For example, for a cluster installed on bare-metal infrastructure with 8 nodes you can configure 150 egress IP addresses. However, if a public cloud provider limits IP address capacity to 10 IP addresses per node, the total number of assignable IP addresses is only 80. To achieve the same IP address capacity in this example cloud provider, you would need to allocate 7 additional nodes.
+While the Egress IPs capability manages the IP address capacity per node, it is important to plan for this constraint in your deployments. For example, if a public cloud provider limits IP address capacity to 10 IP addresses per node and there are 8 nodes in the cluster, the total number of assignable IP addresses is only 80. To increase this to 150 IP addresses per node, you would need to allocate 7 additional nodes.
```

Comment:
Can reword to something similar as below for simplicity. Need to fix the future tense though.

```suggestion
While the Egress IPs capability manages the IP address capacity per node, it is important to plan for this constraint in your deployments. For example, if a public cloud provider limits IP address capacity to 10 IP addresses per node, for a cluster with 8 nodes, the total number of assignable IP addresses is only 80. To increase this to 150 IP addresses per node, you must allocate 7 additional nodes.
```